ocp_resources/utils.py

```diff
@@ -276,3 +277,15 @@ def _return_resource(_resource, _check_exists, _msg):
                     _check_exists=check_exists,
                     _msg=skip_create_warn_msg,
                 )
+
+
+def wait_status_null(vm, status, timeout):
+    LOGGER.info(f"Wait for {vm.kind} {vm.name} status {status} to be null")
```

Comment:
```suggestion
    LOGGER.info(f"Wait for {vm.kind} {vm.name} status {status} to be None")    
```
get(status) return None if not status

Reply:
done